open-craft/edx-platform
# SE-2170 Port changes not yet merged to upstream into our juniper release branch

This branch starts with the tagged upstream `open-release/juniper.2`, and adds in the code drift that was present in `opencraft-release/ironwood.2` which has not yet been merged upstream.

For PRs where there was only a single commit, I did a simple cherry-pick:

```bash
git cherry-pick -x  branch-from-open-pr
```

But where there were many commits on the PR, I didn't want to clutter up our repo. So I squashed and rebased on juniper, and then cherry-picked that commit here.

**Testing instructions**

Sandbox TBD

For each commit:

* Check the diff against the upstream PR (linked in the commit message)
* Test the change on the sandbox using the upstream PR testing instructions

**Reviewer**


CC @lgp171188
